dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.reactivestreams-1.0.feature

```diff
@@ -1,6 +1,6 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0
 singleton=true
--bundles=com.ibm.websphere.org.reactivestreams.reactive-streams.1.0; location:="dev/api/spec/,lib/"; mavenCoordinates="org.reactivestreams:reactive-streams:1.0.2"
+-bundles=com.ibm.websphere.org.reactivestreams.reactive-streams.1.0; location:="dev/api/third-party/,lib/"; mavenCoordinates="org.reactivestreams:reactive-streams:1.0.2"
```

Comment:
This change required a change to 
/libertyGit/open-liberty/dev/com.ibm.websphere.appserver.features/visibility/public/mpReactiveStreams-1.0/com.ibm.websphere.appserver.mpReactiveStreams-1.0.feature
to  org.eclipse.microprofile.reactive.streams.spi; type="stable", \
  org.reactivestreams; type="third-party";